src/store/index.js

```diff
@@ -180,11 +194,19 @@ export default new Vuex.Store({
           // convert to Map
           .map(role => [role.id, role])
       );
+
+      // update travelers by edition list
```

Comment:
```suggestion
      // update extraTravelers map to only show travelers not in this script
```
Comment seemed a bit confusing to me. 😁